docs/specs/toc.yml

```diff
@@ -486,6 +486,41 @@ outputs:
   docs/f2/toc.json:
   .errors.log: |
     ["warning","file-not-found","Invalid file link: 'b.md'.","docs/f2/TOC.md",3,4]
+  .dependencymap.json: |
+    {
+        "dependencies": {
+            "docs/f1/TOC.md": [
+                {
+                    "source": "docs/f1/b.md",
+                    "type": "link"
+                }
+            ],
+            "docs/f2/TOC.md": [
+                {
+                    "source": "docs/f2/a.md",
+                    "type": "link"
+                }
+            ],
+            "docs/TOC.md": [
+                {
+                    "source": "docs/f1/b.md",
+                    "type": "link"
+                },
+                {
+                    "source": "docs/f1/TOC.md",
+                    "type": "tocInclusion"
+                },
+                {
+                    "source": "docs/f2/a.md",
```

Comment:
Not all the dependency in referenced TOC is transit. Actually `toc.md` does not depend on `f1/a.md`.

Reply:
good point, seems we need know it's referencing a folder or referencing a toc file